### PR TITLE
scatman is so fun! :+1:

### PR DESCRIPTION
**Pull request by kbjarkefur**
Scatman should be a part of this collection of amazing lyrics!

### CODE REVIEW

File: lyrics/pop/scatmant.txt [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Ski-bi dibby dib yo da dub dub
+Yo da dub dub
+Ski-bi dibby dib yo da dub dub
+Yo da dub dub
+(I'm the Scatman)
```
Comment (kbjarkefur) [inline lyrics/pop/scatmant.txt]: Youre Scatman, right?

Comment (luizaandrade) [reply lyrics/pop/scatmant.txt]: I'm certainly not

Comment (luizaandrade) [inline lyrics/pop/scatmant.txt]:
```suggestion
(Kris is the Scatman)
```

Comment (kbjarkefur) [reply lyrics/pop/scatmant.txt]: ok then!

**Pull request merged**